### PR TITLE
Including information about angular upgrade exception

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Includes more information about an exception when upgrading webapp's angular and a tip to fix it. Also, more information about upgrading lodash library.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @esthermmoturi, can you please review? Thanks

File: content/en/contribute/code/core/update-dependencies.md [hunk 1]
```diff
@@ -51,12 +52,44 @@ Then for each folder go through these steps.
   ```
 
 - Make sure the version of `api/enketo-xslt` is the same as `webapp/enketo-core/enketo-transformer/enketo-xslt`.
-- Don't update `lodash/core` since 4.17.21 is failing, this is an [open issue](https://github.com/lodash/lodash/issues/4904) in their repository.
-- Don't update `helmet` since it is not supporting NodeJS 8 anymore. Its minimum is NodeJS 10+.
-- If you have trouble upgrading any other dependency and you think it'll be challenging to fix it then raise a new issue to upgrade just that dependency. Don't hold up all the other upgrades you've made.
+- Don't update `lodash` to version `4.17.21`, there's a runtime issue when importing `lodash/core` in Admin app:
```
Comment (esthermmoturi) [inline content/en/contribute/code/core/update-dependencies.md:55]: Don't update `lodash` to version `4.17.21`. If you do so, the runtime issue highlighted below will appear when importing `lodash/core` in Admin app:

File: content/en/contribute/code/core/update-dependencies.md [hunk 2]
```diff
@@ -51,12 +52,44 @@ Then for each folder go through these steps.
...
+  ```
+  UnhandledPromiseRejectionWarning: TypeError: stack.get is not a function
+    at equalObjects (/home/user/app/node_modules/lodash/core.js:1303:28)
+    at baseIsEqualDeep (/home/user/app/node_modules/lodash/core.js:729:18)
+    at baseIsEqual (/home/user/app/node_modules/lodash/core.js:664:12)
+    at Function.isEqual (/home/user/app/node_modules/lodash/core.js:2660:12)
+    at estimateCurrentScheduleDimLevel.then.dimLevel (/home/user/app/src/Devices/SmartLight/NASLightDevice.ts:642:24)
+  ```
+  In their code, the issue is here:
```
Comment (esthermmoturi) [inline content/en/contribute/code/core/update-dependencies.md:64]: The issue on the code is here:

File: content/en/contribute/code/core/update-dependencies.md [hunk 3]
```diff
@@ -51,12 +52,44 @@ Then for each folder go through these steps.
...
+  ```
+  if (!_.isEqual(oldProfileConfig, newProfileConfig)) {
+    this.setProfileConfig(newProfileConfig);
+  }
+  ```
+  There was an [open issue](https://github.com/lodash/lodash/issues/4904) in their repository but got deleted (reason unknown). There's [another reported issue](https://github.com/lodash/lodash/issues/4945) explaining the problem, however it hasn't been resolved yet.
```
Comment (esthermmoturi) [inline content/en/contribute/code/core/update-dependencies.md:70]: There was an [open issue](https://github.com/lodash/lodash/issues/4904) in the lodash repository but got deleted (reason unknown). There's [another reported issue](https://github.com/lodash/lodash/issues/4945) explaining the problem, however it hasn't been resolved yet.

**Review by esthermmoturi — CHANGES_REQUESTED** [review]

File: content/en/contribute/code/core/update-dependencies.md (hunk 1 above)
Comment (esthermmoturi) [inline content/en/contribute/code/core/update-dependencies.md:55]: You can add this

**Comment by latin-panda** [comment]
Thanks for the feedback @esthermmoturi, can you please have a look again?

**Review by esthermmoturi — APPROVED** [review]

**Pull request merged**